app/helpers/comments_helper.rb

```diff
@@ -299,9 +301,11 @@ def commenter_id_for_css_classes(comment)
   def css_classes_for_comment(comment)
     return if comment.nil?
 
+    official = "official" if comment&.pseud&.user&.official
```

Comment:
Does this also need to check whether the comment is an anonymous creator comment? Or is the assumption that official users won't have any anonymous works?

Reply:
It doesn't hurt to cover that case, but official accounts should abso-freaking-lutely not have anonymous works.